File: OfflineDPP/app/src/main/java/com/tantecky/offlinedpp/MainActivity.java
package com.tantecky.offlinedpp;

import android.app.Activity;
import android.os.Bundle;
import android.view.Menu;
import android.view.MenuItem;
import android.widget.ArrayAdapter;
import android.widget.ListView;

import java.util.List;


public class MainActivity extends Activity {

    private ListView mLinesListView;

    public ListView getLinesListView() {
        if(mLinesListView == null)
            mLinesListView  = (ListView) findViewById(R.id.linesListView);

        return mLinesListView;
    }


    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        String[] items = new String[]{"prvni", "druhy"};
        ArrayAdapter<String> itemsAdapter =
                new ArrayAdapter<>(this, android.R.layout.simple_list_item_1, items);
        getLinesListView().setAdapter(itemsAdapter);
    }


    @Override
    public boolean onCreateOptionsMenu(Menu menu) {
        // Inflate the menu; this adds items to the action bar if it is present.
        getMenuInflater().inflate(R.menu.menu_main, menu);
        return true;
    }

    @Override
    public boolean onOptionsItemSelected(MenuItem item) {
        // Handle action bar item clicks here. The action bar will
        // automatically handle clicks on the Home/Up button, so long
        // as you specify a parent activity in AndroidManifest.xml.
        int id = item.getItemId();

        //noinspection SimplifiableIfStatement
        if (id == R.id.action_settings) {
            return true;
        }

        return super.onOptionsItemSelected(item);
    }
}

File: FluidX/app/src/main/java/cz/antecky/fluidx/MainActivity.kt
package cz.antecky.fluidx

import android.app.Activity
import android.opengl.GLSurfaceView
import android.os.Bundle
import android.util.Log

class MainActivity : Activity() {
    private lateinit var glView: GLSurfaceView

    public override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)

        // Create a GLSurfaceView instance and set it
        // as the ContentView for this Activity.
        glView = MyGLSurfaceView(this)
        setContentView(glView)
        Log.d(this::class.qualifiedName, "onCreate")
    }
}